intermittent_failures.md

```diff
@@ -0,0 +1,95 @@
+
+In a sense failures in this pipeline are 'expected' and are only used to collect data, not to prevent a regression of functionality. This data should be informing an in-progress fix, to evaluate whether additional work needs to be done. Tests left active in the Sandbox suite for longer than one month will be disabled to save on infrastructure costs.
+
+For information on changing suite registration, please refer to the O3DE Test Onboarding Guide (document not yet created)
+
+## Troubleshooting
+
+### Help! Is my Test Intermittent?
+
+Sometimes it is unclear whether a failure is intermittent. It can also be unclear if the test is detecting an intermittent bug, or if the test itself creates an intermittent result. Here are recommended steps to clarify and isolate the failure:
+
+#### Step 1: Identify
+
+First clarify what failed by reading through execution logs to find errors. If the failure occurred in Jenkins, open the BlueOcean UI and click on the red nodes. If you are executing from a command line interface you may want to save output to a file using the pipe `|` operator:
+`cmake --build build/windows_vs2019 --target Editor --config profile -- /m | build_output.txt`
+`ctest -C profile -L SUITE_main --no-tests=error | test_output.txt`
+
+If the failure occurred in a build, there should be compiler or linker errors displayed.
+
+If a test has failed, there should be a re-run command listed in the output to execute only the failed tests.
+
+#### Step 2: Confirm
+
+If you suspect this failure is intermittent, gather more data to prove and clarify its intermittent behavior:
+
+- Clean your workspace, to test for incremental cache issues, with a command such as `git clean -df`
+- Execute the failing step at least five times to measure how often it fails
+- Isolate a single test using [GoogleTest filtering](https://github.com/google/googletest/blob/master/docs/advanced.md#running-a-subset-of-the-tests) or [PyTest filtering](https://docs.pytest.org/en/6.2.x/usage.html#specifying-tests-selecting-tests)
+- For more data on order-dependent failures, randomize the order of the tests with [GoogleTest shuffle](https://github.com/google/googletest/blob/master/docs/advanced.md#shuffling-the-tests) or [PyTest-random-order](https://github.com/jbasko/pytest-random-order)
+
+For historical health of individual tests in Jekins runs, refer to the Test Metrics Dashboard (currently unavailable)
+
+#### Step 3: Document
+
+After clarifying the failure reason and reproduction rate, create a new [GitHub Issue](https://github.com/o3de/o3de/issues/new/choose) to track a fix. To help raise awareness and priority on the issue, reach out over [Discord](https://discord.gg/p3padwr58u)
```

Comment:
This is worth repeating here..."- Add a label for the SIG which maintains that feature 'sig/XYZ' ***OR*** use label 'needs-sig'"